.migration-backup/ui-components/app/(management)/dashboard.tsx

```
@@ -0,0 +1,519 @@
+import React, { useState, useEffect, useCallback } from 'react';
```

ESLint

[error] `react` import should occur after import of `expo-router`

(import/order)